src/api/fetchRecipes.js

```diff
@@ -4,21 +4,21 @@ const apiID = process.env.REACT_APP_RECIPES_APP_ID;
 const apiKey = process.env.REACT_APP_RECIPES_APP_KEY;
 
 const recipesUrl = "/recipes/v2";
-const ingr = "";
 
-const fetchRecieps = async () => {
+const fetchRecipes = async (ingredients, calories) => {
```

Comment:
It should be better to pass object as arguments
`const fetchRecipes = async ({ ingredients, calories })`